### DISCUSSION TITLE
2.6.11 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#2908] [讨论]
· Mod 管理功能优化
　　将会显示 Mod 的中文名、图标、标签等信息（#158，@WForst-Breeze）
　　查看 Mod 信息时会跳转到其下载详情页面（#1336，@qizri）
　　搜索 Mod 文件时支持搜索中文名与版本号
　　Mod 文件的禁用、错误状态将由图标右下角的指示标记显示
　　压缩管理卡片的占用空间，只保留功能按钮
　　在已全选时再次点击全选按钮会取消全选
　　加载进度会更平滑地显示，而非几乎不变
　　选中的 Mod 将具有底色，便于区分
· 列表项界面与动画优化
　　增加列表右侧小图标按钮的尺寸，让颜色更明显，以此提高醒目度
　　描述不再为右侧小图标留出空白，改为在鼠标指向时播放缩短的动画
　　修改“初始化”按钮的图标，以与“刷新”按钮的图标粗细吻合
· 启用 SSL 证书有效性校验（#2767，@litwak913）
· 添加樱花披风的名称汉化（#2845，@596wjr）
· 迁移 Fandom Wiki 链接（#2864、LTCatt/PCL2Help#337，by @zkitefly 和 @idhaname）
· 帮助库：优化光影、整合包安装教程（LTCatt/PCL2Help#336，by @WForst-Breeze）
· 将所有 http 协议更改为 https 协议，提高安全性
· 当连接 CurseForge 或 Modrinth 失败时显示警告信息
· 新主页预设：历史中的今天（by @hsuchenghao）
· 修复了以下 13 个 Bug
　　PCL 版本号最后有意料外的 0（#2813）
　　特定情况下取消启动导致版本设置中的 Java 被重置（#2820，@MM-Rabbit）
　　音乐播放时删除其他音乐导致出错（#2821，@JJJJGGGG1）
　　罕见情况下播放音乐报错：索引超出范围（#2822，@JingHai-Lingyun）
　　部分声音库相关的崩溃会给出错误的处理建议（#2861，@23998886688RdHy）
　　清空背景图片不会删除子文件夹下的文件（#2867，@LSGtorch）
　　在下载 Java 时手动搜索 Java 会导致获取失败（#2872，@JingHai-Lingyun）
　　无法下载 CurseForge 整合包中的极个别 Mod（#2899，@xiaobian663）
　　在资源下载的左边栏点击刷新按钮时，没有清理本地的缓存信息
　　重复获取了部分 CurseForge Mod 的文件信息，导致加载变慢
　　资源下载详情页顶部栏的描述 Tooltip 无法在鼠标指向时展开
　　微软登录步骤 2 失败时不会尝试重试
　　加载空 Mod 文件后不会释放文件权限

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.6.11'>2.6.11</a>.</em>

**Comment by MoYuan-CN** [comment]
还是头一回在一次更新里贡献两个反馈呢（开心

**Reply by MoYuan-CN** [reply]
然后这回更新的彩蛋是：忘记写更新日志的龙猫~
![a2e16a26ae8a71fe522e34cc7e97391a](https://github.com/Hex-Dragon/PCL2/assets/109946426/6124e16e-9d3a-445c-ae42-f2b256a5177d)

**Reply by SqrtTan** [reply]
~众所周知，lm忙着玩王泪~

**Reply by MoYuan-CN** [reply]
哈哈哈

**Comment by xiaobian663** [comment]
我的启动器没发更新
![QQ截图20231130142444](https://github.com/Hex-Dragon/PCL2/assets/103417178/22cb550e-556e-4804-8556-99e5dde1e205)

**Reply by LTCatt** [reply]
正式版尚未推送

**Reply by MoYuan-CN** [reply]
哦对 这是 1000（指正式版
~~给这茬忘了~~
